### PR TITLE
[PERF] inlines _initializeListeners in meta

### LINKED ISSUE #15171: We should just inline _initializeListeners in meta
No point in keeping this separate we should just inline it, rather then waiting for the JIT to doit.

https://github.com/emberjs/ember.js/blob/master/packages/ember-metal/lib/meta.js#L127

### PR DESCRIPTION
**Pull request by tylerturdenpants**
This fixes #15171, per @stefanpenner

### CODE REVIEW

File: packages/ember-metal/lib/meta.js [hunk 1]
```diff
@@ -124,7 +124,9 @@ export class Meta {
       }
     }
 
-    this._initializeListeners();
+	this._listeners = undefined;
```
Comment (stefanpenner) [inline packages/ember-metal/lib/meta.js]: Mind fixing up the indentation?

**Review by stefanpenner — CHANGES_REQUESTED** [review]
nice!!

Let one nit-pick though, but then we can land.

File: packages/ember-metal/lib/meta.js (hunk 1 above)
Comment (tylerturdenpants) [inline packages/ember-metal/lib/meta.js]: sorry, my editor settings!  will fix

**Comment by tylerturdenpants** [comment]
@stefanpenner forgive me, i'm trying to figure out the indentation

**Comment by tylerturdenpants** [comment]
fails saucelab tests, not sure why.

**Review by stefanpenner — APPROVED** [review]

**Comment by stefanpenner** [comment]
@tylerturdenpants our sauce labs tests tend to be somewhat brittle, restarting them now. Otherwise this looks good!

**Pull request merged**